repos/system_upgrade/common/actors/udev/udevadminfo/libraries/udevadminfo.py

```diff
@@ -0,0 +1,11 @@
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import UdevAdmInfoData
+
+
+def process():
+    try:
+        out = run(['udevadm', 'info', '-e'])['stdout']
+    except CalledProcessError:
+        api.current_logger().error("Failed to run 'udevadm info -e'")
+        out = ''
```

Comment:
@tomasfratrik  that's actually invalid. It changes the behaviour like in the way "nothing wrong happened, continue". try-except is useful, but it's expected to actually raise StopActorExecutionError with the proper message that we cannot obtain required information about devices from udev. In details, you could add the original error msg.

Reply:
Okay I see, I'll try to fix it